benchmarks/graphics_pipeline/main.cpp

```diff
@@ -299,10 +327,10 @@ void ProjApp::LoadMaterial(
 
             // FIXME: assuming all primitives provides POSITION, UV, NORMAL and TANGENT. Might not be the case.
             gpCreateInfo.vertexInputState.bindingCount      = 4;
-            gpCreateInfo.vertexInputState.bindings[0]       = mPrimitives[0].mesh->GetDerivedVertexBindings()[0];
-            gpCreateInfo.vertexInputState.bindings[1]       = mPrimitives[0].mesh->GetDerivedVertexBindings()[1];
-            gpCreateInfo.vertexInputState.bindings[2]       = mPrimitives[0].mesh->GetDerivedVertexBindings()[2];
-            gpCreateInfo.vertexInputState.bindings[3]       = mPrimitives[0].mesh->GetDerivedVertexBindings()[3];
+            gpCreateInfo.vertexInputState.bindings[0]       = mScenes[0].primitives[0].mesh->GetDerivedVertexBindings()[0];
```

Comment:
Should this take into account the scene? Or the FIXME above applies where we assume all primitives in all scenes are the same

Reply:
Yes, FIXME applies. Handling varying attributes means we would have to add logic to generate tangents, etc, and would make the application more complex.
For now, don't seem to be an issue.